src/dataset-builder/ConceptSetCreator.test.ts

```diff
@@ -27,7 +28,7 @@ describe('ConceptSetCreator', () => {
     return { conceptSetUpdater, onStateChange };
   };
 
-  const rootConcept = dataset!.snapshotBuilderSettings!.domainOptions[0].root;
+  const rootConcept = snapshotBuilderSettings!.domainOptions[0].root;
```

Comment:
I don't think the `!` is needed here
```suggestion
  const rootConcept = snapshotBuilderSettings.domainOptions[0].root;
```